Python/Product/Debugger/DebugAdapterProcess.cs

```diff
@@ -178,6 +179,24 @@ sealed class DebugAdapterProcess :
             if (_stream != null) {
                 _stream.Dispose();
             }
+
+            if (_process.ExitCode == 126) {
+                // 126 : ERROR_MOD_NOT_FOUND
+                // This error code is returned only for the experimental debugger. MessageBox must be
+                // bound to the VS Main window otherwise it can be hidden behind the main window and the 
+                // user may not see it.
+                MessageBox.Show(
+                    new VSWin32Window(Process.GetCurrentProcess().MainWindowHandle),
+                    Strings.ImportPtvsdFailedMessage,
+                    Strings.ImportPtvsdFailedTitle,
+                    MessageBoxButtons.OK,
+                    MessageBoxIcon.Error);
```

Comment:
I'm okay with using a plain message box right now, as we intend to remove the need for it fairly soon (though we'll leave it here in case someone runs into a corrupt install).

A permanent error like this would be better as a TaskDialog that can trigger the action (in this case, install `--pre ptvsd` into the current environment).